pallets/crowdloan-claim/src/lib.rs

```diff
@@ -751,6 +756,22 @@ impl<T: Config> Pallet<T> {
 		T::PalletId::get().into_account()
 	}
 
+	/// Build a sorted hash of two given hash values.
+	///
+	/// Hash a:b if a < b, else b:a. Uses the runtime module's hasher.
+	pub fn sorted_hash_of(a: &T::Hash, b: &T::Hash) -> T::Hash {
```

Comment:
we have the proofs library for proof verification. Can you use that ?

Reply:
Isn't that part of the pallet claims?

Reply:
Nevermind I found it.

Reply:
@vedhavyas and @mikiquantum I would prefer to have this duplicate code. 
I would a Hashing algorithm to implement `Hasher` trait, as we are not using the substrate I can not use the system hashing here. Would be kind overkill in my opinion just for using this one function. 

WDYT?

Reply:
This is thee change I was suggesting https://github.com/centrifuge/centrifuge-chain/blob/parachain/libs/proofs/src/mock.rs#L15

Not sure I completely follow you